features/lwipstack/lwipopts.h

```diff
@@ -83,9 +83,7 @@
 
 #define SYS_LIGHTWEIGHT_PROT        1
 
-#ifndef LWIP_RAW
-#define LWIP_RAW                    0
-#endif
+#define LWIP_RAW                    1
```

Comment:
These changes were added for PSOC6 testing but it is not clear if they should remain here as they affect all targets.
@vmedcy @morser499

Reply:
It should not, this PR can be dependent on it (will be treated like that if required). I already mentioned few other issues like this, please create separate PR. They will get much faster in and easier to review.

Reply:
LWIP_RAW change moved to https://github.com/ARMmbed/mbed-os/pull/10978/files